homeassistant/components/deconz/__init__.py

```diff
@@ -84,3 +73,33 @@ async def async_update_master_gateway(hass, config_entry):
     options = {**config_entry.options, CONF_MASTER_GATEWAY: master}
 
     hass.config_entries.async_update_entry(config_entry, options=options)
+
+
+async def async_migrate_entry(hass, config_entry) -> bool:
+    """Migrate old entry."""
+    LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    # Update unique ID entities based on deCONZ groups
+    if config_entry.version == 1:
+        old_unique_id = config_entry.data.get(CONF_GROUP_ID_BASE)
+        if old_unique_id:
+            # if old_unique_id := config_entry.data.get(CONF_GROUP_ID_BASE):
```

Comment:
Please remove commented code.

Reply:
This was kept in order to get passed commit scripts, flake8 didn't allow it even though it should. Hopefully I can restore to use the walrus operator, otherwise I will remove it.